source/_integrations/sunsynkweb.markdown

```diff
@@ -0,0 +1,64 @@
+
+---
+title: Sunsynk web
+description: Home Assistant integration to gather statistics from [Sunsynk connect](http://sunsynk.org) REST API
+ha_category:
+
+- Energy
+- Sensor
+ha_release: TBD
+ha_iot_class: Cloud Polling
+ha_codeowners:
+- '@francoisverbeek'
+ha_domain: sunsynkweb
+ha_platforms:
+- sensor
+ha_config_flow: true
+ha_integration_type: integration
+
+---
+
+The **Sunsynk web** {% term integration %} gathers statistics from [Sunsynk connect](http://sunsynk.org). It exposes real-time and historical information.
+
+This will log into your Sunsynk account and will collate the information of any plants or inverters and present the main sensors for the plant, real-time and historical.
+
+## Prerequisites
+
+To use this integration you will need a sunsynk-branded e-linter [wifi dongle](https://www.sunsynk.org/logger-support) for your inverter(s) and a [Sunsynk connect](Sunsynk.net) account. This will typically be created by your installer on your behalf. The account needs at least the 'visitor' permissions.
```

Comment:
```suggestion
To use this integration, you will need a sunsynk-branded e-linter [wifi dongle](https://www.sunsynk.org/logger-support) for your inverter(s) and a [Sunsynk connect](Sunsynk.net) account. This will typically be created by your installer on your behalf. The account needs at least the 'visitor' permissions.
```